documentation/modules/proc-configuring-kafka-user.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-user-operator.adoc
+
+[id='proc-configuring-kafka-user-{context}']
+= Configuring a Kafka user
+
+Use the properties of the `KafkaUser` resource to configure a Kafka user.
+
+You can use `kubectl apply` to create or modify users, and `kubectl delete` to delete existing users.
+
+* `kubectl apply -f _<user-config-file>_`
+* `kubectl delete KafkaUser _<user-name>_`
+
+When you create a user, you must configure the authentication and authorization mechanisms required when connecting to Kafka brokers using a listener.
+
+This procedure shows how a user is created with TLS authentication.
+The authentication required depends on the xref:assembly-kafka-broker-listener-authentication-deployment-configuration-kafka[type of authentication configured for the Kafka broker listener].
```

Comment:
It does and it doesn't .... the user will not get any error when such authentication is not enabled in the broker. There is no check for that. So this might be a bit confusing.

Reply:
The user won't get an error but anyway user authentication depends on the authentication settings in Kafka, of course, you cannot authenticate a user via TLS if it's not enabled in the Kafka configuration. I think that this sentence is appropriate here.

Reply:
Maybe we could add a note:

> NOTE: Authentication between Kafka users and Kafka brokers depends on the authentication settings for each. For example, it is not possible to authenticate a user through TLS if it is not also enabled in the Kafka configuration.

Not sure if this is too much or makes things clearer?

Reply:
I think this is good. I would also add similar note on authorization since that seems to be a common mistake among users that they do not enable it and wonder why it does not work.

Reply:
Yes it's good for me as well.

Reply:
:+1: